config/kibana.yml

```diff
@@ -181,3 +181,6 @@
 # Maximum number of documents loaded by each shard to generate autocomplete suggestions.
 # This value must be a whole number greater than zero. Defaults to 100_000
 #unifiedSearch.autocomplete.valueSuggestions.terminateAfter: 100000
+
+
+uiSettings.experimental.themeSwitcherEnabled: true
```

Comment:
```suggestion
```

Reply:
Thanks but do you know if the designers have finished testing or if the previous deployment is still available?